packages/interface/src/components/complementary-area/index.js

```diff
@@ -150,6 +153,16 @@ function ComplementaryArea( {
 					/>
 				</PinnedItems>
 			) }
+			{ __unstableInitSource !== 'core' && isPinnable && (
```

Comment:
mmm,  so "core" is a way to opt-out of the menu item?

Reply:
Do you think plugins could benefit this (regardless of whether they provide  their  own menu item). Meaning, do you think it should be an explicit `noMenuItem` prop?

Reply:
Yes 🙈